### PR TITLE
elastic/obs-infraobs-integrations: Address skipped validation SVR00004

### PR DESCRIPTION
**Pull request by shmsr**
Tip: How to use the inliner script? Follow [this](https://github.com/elastic/visualizations_integrations_tools#inliner-usage). Also read the following:

> Set this to your environment (e.g., prepend when executing the inliner script) `NODE_TLS_REJECT_UNAUTHORIZED='0'` to bypass certificate verification for TLS connections and HTTPS requests as elastic-package deploys the stack with TLS on by default.

## Proposed commit message

Used https://github.com/elastic/visualizations_integrations_tools/blob/master/inline.js to inline "by reference" viz.

- [x] apache_spark
- [x] azure_functions
- [x] hadoop
- [x] iis
- [ ] oracle_weblogic
- [x] spring_boot
- [x] traefik
- [x] websphere_application_server

Couldn't migrate other packages 11 packages out of 19 packages with SVR00004 as inliner script does not support inlining saved searches because the script was developed before users could have by-value (inline) saved searches in Kibana dashboards. So those packages have references to "search" and hence SVR00004 couldn't be addressed.

Thanks to Drew for creating this related issue to update the inliner script: https://github.com/elastic/visualizations_integrations_tools/issues/2

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

Here is a list of packages that have SVR00004 with the kibana.version:

```
activemq == ^8.8.0 
apache_spark == ^8.8.0
apache_tomcat == ^8.8.0
azure_functions == ^8.8.1
cassandra == ^8.8.0
hadoop == ^8.10.2
iis == ^8.8.0
kafka == ^8.8.0
microsoft_sqlserver == ^8.10.2
mongodb == ^8.8.0
mysql == ^8.8.2
nginx == ^8.8.0
oracle_weblogic == ^8.8.0
postgresql == ^8.8.0
redis == ^8.10.2
spring_boot == ^8.9.0
system == ^8.10.2
traefik == ^8.9.0
websphere_application_server == ^8.3.0
```

- [x] Note that, using the inliner script will make the dashboards incompatible with earlier versions of the stack. For example, if the inliner script is used with 8.8 stack then the dashboard generated will be compatible with stack 8.8 and above.
- [x] So, all packages with SVR00004 can be migrated with 8.8.0 stack as the kibana.version is either 8.8.0 or above except `websphere_application_server` (8.3.0).

> * For all packages, except `websphere_application_server`; stack used for inliner script: v8.8.0
> * For `websphere_application_server`, stack used for inliner script: v8.3.0

## How to test this PR locally

Review should include loading the dashboard into an instance with data to make sure everything is displayed properly.

## Related issues

- https://github.com/elastic/integrations/issues/8452
- https://github.com/elastic/integrations/issues/8028

### CODE REVIEW

**Comment by shmsr** [comment]
Please read the full PR description before reviewing.

**Comment by lalit-satapathy** [comment]
CC: @harnish-elastic @aliabbas-elastic to review the dashboard changes as part of the pass by value is correctly being added.

**Review by aliabbas-elastic — APPROVED** [review]
LGTM for the changes. Please refer [here](https://github.com/elastic/obs-infraobs-team/issues/1263) for testing details

**Comment by mrodm** [comment]
Hi @shmsr, please update your branch with the latest contents from main branch. There was an important PR merged updating the CI pipelines. Thanks!

**Comment by lalit-satapathy** [comment]
@shmsr, 

What is pending on this as the [validation](https://github.com/elastic/obs-infraobs-team/issues/1263#issuecomment-1840362846) is complete?

**Comment by shmsr** [comment]
> @shmsr,
> 
> What is pending on this as the [validation](https://github.com/elastic/obs-infraobs-team/issues/1263#issuecomment-1840362846) is complete?

I have to resolve the merge conflicts. I'll do it ASAP.

**Comment by shmsr** [comment]
@harnish-elastic @aliabbas-elastic I have resolved the merge conflicts. I had to inline viz in springboot from the checkout of main brain as dashboard changes had been made in the last couple of months. Otherwise, for other packages only manifest and changelog yaml's are touched.

**Comment by shmsr** [comment]
Fixed merge conflicts.

**Review by devamanv — APPROVED** [review]

File: packages/oracle_weblogic/manifest.yml
Comment (harnish-elastic) [inline packages/oracle_weblogic/manifest.yml:1]: Please refer to this [comment](https://github.com/elastic/integrations/issues/8938#issuecomment-1907554969). The saved search will throw error if it is inlined in the Kibana <`8.10`.

Comment (harnish-elastic) [reply packages/oracle_weblogic/manifest.yml:1]: Error is mentioned [here](https://github.com/elastic/integrations/issues/8938#issuecomment-1905712911).

**Review by harnish-elastic — CHANGES_REQUESTED** [review]

**Comment by shmsr** [comment]
As latest changes in oracle_weblogic includes saved searches; inlining them could be done for 8.10 stack. So leaving out oracle_weblogic.

cc: @harnish-elastic

**Pull request merged**